tests/testthat/test-5-methods.R

```
@@ -3,10 +3,15 @@ skip_on_cran()
 
 # the summary tables and print functions are imported from BayesTools and tested henceforth
 # test objects - assuming that the fit function worked properly
+temp_fits_dir <- Sys.getenv("ROBMA_TEST_FITS_DIR")
+if (temp_fits_dir == "" || !dir.exists(temp_fits_dir)) {
```

Instead of skipping, throw an error that the fitted objects are missing.